trmnl.yaml

```diff
@@ -22,7 +22,9 @@ ota:
 
 wifi:
   # Set up a wifi access point using the device name above
+  # SSID needs to be static so the QR code shown on screen works
```

Comment:
The QR code value can be set and recalculated at runtime to include the complete dynamic ssid if desired.